### ISSUE TITLE
proximal interphalangeal (PIP) joint arthritis

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10712]
**Preferred term label:**
proximal interphalangeal (PIP) joint arthritis on imaging

**Synonyms**

**Definition (free text, please give PubMed ID)**
PIP joint space narrowing, subchondral cysts, and can have osteophytes

Note: osteophytes also can be palpated as Bouchard nodes, but this is a mention of features seen on imaging.

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Proximal interphalangeal (PIP) joint arthritis (it has several other clinical findings before confirming using imaging).

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001392
- label: Proximal interphalangeal joint arthritis
- parents: [Arthritis (HP:0001369)](https://hpo.jax.org/app/browse/term/HP:0001369)
- pmid: PMID:31312517
- def: Inflammation and deterioration of the proximal interphalangeal (PIP) joints. Radiographically, PIP arthritis can be characterized by PIP joint space narrowing, subchondral cysts, and osteophytes.
- comment: PIP arthritis manifests clinically with swollen, tender PIP joints, with a more diffuse, swollen appearance and a fusiform joint contour. Joint stiffness is almost always present and often correlates with the degree of swelling.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10712
- synonyms:
